index.html

```diff
@@ -289,6 +289,25 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact-me">
+        <header class="contact-me-header">
+          <h1>
+            I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h1>
+        </header>
+        <form onsubmit="validate()" action="https://formspree.io/f/mbjqyrqj" method="POST">
+          <fieldset tabindex="0">
+            <input type="text" name="name"
+              id="fullname" maxlength="30" placeholder="Full name">
+            <!-- <input type="email" required name="_replyto" id="email"> -->
+            <input type="email" placeholder="Email address" required name="_replyto" id="email">
+            <textarea name="tarea" maxlength="500" placeholder="Enter text here"></textarea>
+          </fieldset>
```

Comment:
- [ ] REQUIRED: Let's add a validation to restrict the form from submitting unless all fields have been properly filled out. This is easily achieved by adding the [`required` ](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required) attribute to essential form inputs. In this case, all inputs should have the required attribute.